test/DynamoCoreWpfTests/SplashScreenTests.cs

```diff
@@ -1,33 +1,50 @@
 
 
+using DynamoUtilities;
 using NUnit.Framework;
 
 namespace DynamoCoreWpfTests
 {
     [TestFixture]
     internal class SplashScreenTests
     {
+        [SetUp]
+        public void SetUp()
+        {
+            TestUtilities.WebView2Stamp = TestContext.CurrentContext.Test.Name;
```

Comment:
yeah I guess it would just be nice if instead of sprinkling this logic all over for any UI test it was just handled in the DynamoWebView2 wrapper.

Reply:
I could try it. But it would mean to have more testing logic in runtime code. It would probably involve reflection and asembly checking..

Reply:
let's leave that potential refactor for later, for now it's just one static string.